### DISCUSSION TITLE
如何设置更目录使用密码访问？

### DISCUSSION THREAD

**Discussion by Kaiyuan** [spencerwooo/onedrive-vercel-index#652] [Q&A] [answered]
在配置文件那里设置 
```
protectedRoutes: ['/'],
```
提示错误
```
{"error":"Internal server error."}
```

已经建立了 `.password` 文件

**Comment by myl7** [comment, marked as answer]
可以先尝试一下清除缓存（通过触发一次 Vercel deploy 实现）试试

**Reply by Kaiyuan** [reply]
试过了，还是这样。需要删除数据库重新建立吗？

**Reply by myl7** [reply]
不需要清数据库，因为数据库和密码保护没有关系

你的密码是全数字的吗？全数字密码有 bug 需要 apply 一个修复的 PR

**Reply by Kaiyuan** [reply]
我是纯数字密码，现在清了数据库之后再重新授权，提示不是网站所有者...
![屏幕截图 2022-05-23 100523](https://user-images.githubusercontent.com/1176383/169729889-df2a62bf-244c-442f-a143-f72c6fd93f0b.png)

**Reply by myl7** [reply]
纯数字密码需要 apply #638 

提示不是网站所有者的话，改一下配置文件中的 userPrincipleName 这配置项为你的 OneDrive 用户名即可

**Comment by Kaiyuan** [comment]
密码改为字母加数字就正常了。
